dynamixel_hardware/src/dynamixel_hardware.cpp

```diff
@@ -38,6 +38,15 @@ constexpr const char * kPresentVelocityItem = "Present_Velocity";
 constexpr const char * kPresentSpeedItem = "Present_Speed";
 constexpr const char * kPresentCurrentItem = "Present_Current";
 constexpr const char * kPresentLoadItem = "Present_Load";
+constexpr const char* const extraJointParameters[] = {
```

Comment:
Please follow the Google Coding Rule.

```suggestion
constexpr const char* const KExtraJointParameters[] = {
```

Reply:
Sure! :+1: I've updated my PR now.